src/JustSaying/AwsTools/QueueCreation/SubscriptionType.cs

```diff
@@ -0,0 +1,8 @@
+namespace JustSaying.AwsTools.QueueCreation
+{
+    public enum SubscriptionType
```

Comment:
`///` docs

Reply:
what should it say?

Reply:
_"An enumeration of subscription types."_? Maybe add some more depending on what it's used to do by these changes?

Public members in libraries should have `///` documentation on them for their users so they show up in things like Intellisense in Visual Studio, particularly if there's no formal documentation anywhere else.

Reply:
Yeah I don't mind adding docs, I just can't find anything clear about what this does. Sure I can just put that.

Reply:
Also needs some `///` docs for the values.